CONTRIBUTING.md

```diff
@@ -250,6 +250,14 @@ In most cases, they should be used instead of (deprecated) `bson.D.Map()`,
 The bar for adding new helpers is very high.
 Please check all existing ones.
 
+If there's a need to use any large number to test corner cases, 
+we create constants for them with explanation what do they represent, and refer to them.
+For example:
+
+```
+const doubleMaxPrec = float64(1<<53 - 1) // 9007199254740991.0:    largest double values that could be represented as integer exactly
```

Comment:
In our code base we have mixture of `float64(9007199254740991)` and `9007199254740991.0`. More commonly found is `0.0` and `float64(0)`. @quasilyte identified this as something we can make more consistent via linter. What is our preference? This documentation just reminded me of that :hugs:

Reply:
I guess in simpler cases like `0.0` it seems better to use this notation

Reply:
Okie :+1:

Reply:
I guess no action on this? Perhaps the task of linter if we agree

Reply:
> I guess no action on this?

I made all of the constants to use `float64()`